README.md

```diff
@@ -2,6 +2,49 @@
 ETHOnline 2022 entry by ChainHackers
 https://gamejutsu.app/
 
+# Run locally
+```shell
+yarn install
+```
+add the file ```.env.local``` to the root of the project and in it declare the variable ```NEXT_PUBLIC_ALCHEMY_API_KEY with the key from ALCHEMY_API```
+
+```shell
+yarn generate
+yarn dev
+```
+
+### Development
+```shell
+yarn install
+yarn dev
+```
+
+### Build
+```shell
+yarn install
+yarn build
+```
+
+# Deploy
+Check for a version of
+```shell
+git tag -l
+```
+
+Select the version you are running - `v1.1.20 is a sample version`
+
+```shell
+git tag v1.1.20
+```
+
+Push origin
+```shell
+git push origin v1.1.20
+```
+
+###(up to u) Docker
+???
```

Comment:
Please don't leave the ??? if you don't complete this one, just drop it :)

Reply:
well deleted